apps/full-site-editing/full-site-editing-plugin/starter-page-templates/class-starter-page-templates.php

```diff
@@ -176,6 +176,7 @@ public function enqueue_assets() {
 				'templates'       => array_merge( $default_templates, $vertical_templates ),
 				'vertical'        => $vertical,
 				'segment'         => $segment,
+				'screen'          => get_current_screen(),
```

Comment:
Should we just pass the action here to avoid adding extra info?

Reply:
Yes we should!  especially since get_current_screen() was already assigned to $screen earlier in that function.  Will replace this with ` 'screenAction' => $screen->action`